### PR TITLE
no setuptools-markdown

### PR DESCRIPTION
**Pull request by ocefpaf**
PyPI will soon support markdown files making this dependency unnecessary.
See https://packaging.python.org/specifications/#description-content-type

### CODE REVIEW

**Comment by scls19fr** [comment]
Great news Markdown support with PyPI was a long awaited feature!

https://bitbucket.org/pypa/pypi/issues/148/support-markdown-for-readmes
https://github.com/pypa/pypi-legacy/issues/148
https://github.com/pypa/warehouse/issues/869

but I'm not sure if this feature is live now

moreover I wonder how `Description-Content-Type` should be set using `setuptools`
but `https://github.com/pypa/setuptools/pull/1075` is not merged for now

**Comment by ocefpaf** [comment]
Yep. It will take a while longer :unamused: 

But the use of `setuptools-markdown` here makes it quite hard to package this with external package managers. I am currently "patching" the source to remove it to be able to package it.

TL;DR you don't need to merge this now, or ever :smile:, but I really hope that soon `README.md` will be the standard for `PyPI`.

**Comment by scls19fr** [comment]
According `https://github.com/pypa/setuptools/pull/1075` and `https://github.com/pypa/python-packaging-user-guide/pull/258`

we will probably need to add

    long_description_content_type='text/markdown; charset=UTF-8; variant=CommonMark',

or

    long_description_content_type='text/markdown; charset=UTF-8',

which default to Markdown variant CommonMark

**Comment by ocefpaf** [comment]
Yep. I can add that if you are interested in the PR.

**Comment by scls19fr** [comment]
Yes. I will merge it when Markdown support in setuptools, twine and Warehouse will be merged.

Just send me a reminder if I forget to merge when the full chain will be ready.

**Comment by scls19fr** [comment]
What is current status ?

**Comment by ocefpaf** [comment]
I believe that is done upstream. [Here is the oficial example](https://github.com/pypa/sampleproject/pull/66) and [the issue](https://github.com/pypa/warehouse/issues/869) for the Markdown support.

**Comment by scls19fr** [comment]
LGTM also
